src/Compiler/CompilerUtils.cpp

```diff
@@ -126,6 +126,12 @@ static llvm::cl::opt<std::string> march("march",
     llvm::cl::value_desc("Target a specific architecture type"),
     llvm::cl::cat(OnnxMlirOptions), llvm::cl::ValueRequired);
 
+static llvm::cl::list<accel::Accelerator::Kind> maccel("maccel",
```

Comment:
I did not use a string for the template argument and used the strongly typed enum in the Accelerator class instead. This makes it easy to use a factory method to construct accelerators, see below.